doc/guides/examples.md

```diff
@@ -94,3 +94,16 @@ query hello {
 The query above will only work if the authenticated user has the `admin` role. You can see this rule being applied with the `@hasRole(role: "admin")` directive in [examples/keycloak/server.js](../../examples/keycloak/server.js#L22).
 
 Try change the the role to a made up role and restart the server. Try the sample query again and verify that an error is displayed.
+
+## 3rd party integrations example
+
+`integrations/index.js` shows, how it is easy to external web service services under a GraphQL based API.
```

Comment:
[Suggestion] `integrations/index.js` shows how easy it is to integrate the consumption of a REST API endpoint into your GraphQL resolver.

Reply:
I'll take any suggestion. My English is pretty bad. :)